presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveTransactionalTable.java

```diff
@@ -358,6 +364,42 @@ public void testCreateAcidTable(boolean isPartitioned, BucketingType bucketingTy
         }
     }
 
+    @Test(groups = HIVE_TRANSACTIONAL)
+    public void testFilesForAbortedTransactionsIgnored()
+            throws TException, IOException
+    {
```

Comment:
@losipiuk : removed partition test case, I guess as we are only testing at the delta directory level, so it won't make any difference. thoughts?